src/server/biosecurity/grazing/index.test.js

```diff
@@ -0,0 +1,77 @@
+import { GrazingAnswer } from '../../common/model/answer/grazing/grazing.js'
+import { describePageSnapshot } from '../../common/test-helpers/snapshot-page.js'
+import { roadsAndTracksPage } from '../roads-and-tracks/index.js'
+import { lastGrazedPage } from '../last-grazed/index.js'
+import { grazing, grazingPage, GrazingPage } from './index.js'
+
+const sectionKey = 'biosecurity'
+const question = 'Will the incoming cattle be grazed?'
+const questionKey = 'grazing'
+const view = 'common/model/page/question-page.njk'
+const pageUrl = '/biosecurity/grazing'
+
+describe('GrazingPage', () => {
+  let page
```

Comment:
just `const page = new GrazingPage()`.  We shouldn't be mutating this during the test, and this helps typescript help us.